src/client/src/apps/MainRoute/widgets/blotter/components/Blotter.tsx

```diff
@@ -62,7 +62,8 @@ const getRowClass = ({ data }: { data: Trade }) => {
 }
 
 const Blotter: React.FC<BlotterProps> = props => {
-  const { canPopout, rows } = props
+  let { canPopout, rows } = props
+  rows = rows.map((a, i) => (i === 0 ? { ...a, latest: true } : a))
```

Comment:
What is the impact of this extra step on performance?
Is there a way (1) to use react key attributes to identify the right row (2) if there is no other way to do this, is it possible to enable this code while tests are running only, and not in production?